src/lld/dpdk/infra/dpdk_cli.c

```diff
@@ -999,46 +999,85 @@ cmd_pipeline_build(char **tokens,
 	char *out,
 	size_t out_size)
 {
-	struct pipeline *p = NULL;
-	FILE *spec = NULL;
-	uint32_t err_line;
-	const char *err_msg;
-	int status;
+	struct rte_swx_pipeline *p = NULL;
```

Comment:
Arrange local declaration in Reverse Christmas tree.

Reply:
taken care